src/components/BuilderComponents/stories/HeaderAndMarkDownBlock.stories.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import HeaderAndMarkDownBlock from "../HeaderAndMarkDownBlock";
+
+export default {
+  title: "BuilderComponents/HeaderAndMarkDownBlock",
+  component: HeaderAndMarkDownBlock,
+  argTypes: {
```

Comment:
Storybook can infer the other types based on the PropTypes (text, boolean, radio if you use `PropTypes.oneOfType`) and provides a control to interact with. It can't automatically infer colors so you have to explicitly define it in the argTypes.